example/server/server.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"net"
+	"log"
+	"github.com/google/s2a-go/example/echo"
+	"github.com/google/s2a-go/internal/v2"
+	"google.golang.org/grpc"
+	pb "github.com/google/s2a-go/example/proto/echo_go_proto"
+)
+
+var (
+	listenPort = flag.String("listen_port", ":8080", "Echo service address port.")
+)
+
+func runServer(listenPort *string) {
+	creds, err := v2.NewServerCreds()
+	listener, err := net.Listen("tcp", *listenPort)
+	if err != nil {
+		log.Fatalf("failed to listen on addres %s: %v", *listenPort, err)
+	}
+	s := grpc.NewServer(grpc.Creds(creds))
+	log.Printf("Server: starting gRPC Echo Server at: %s", *listenPort)
+	log.Printf("Server: started listening on: %s", *listenPort)
```

Comment:
Should we omit one of these 2 print statements, since they're essentially doing the same thing?